mne/viz/topomap.py

```diff
@@ -1134,19 +1135,34 @@ def plot_evoked_topomap(evoked, times=None, ch_type=None, layout=None,
 
     if isinstance(axes, plt.Axes):
         axes = [axes]
-    if times is None:
+
+    if times == "peaks":
+        from scipy.signal import argrelmax
+        gfp = evoked.data.std(axis=0)
+        order = int(evoked.info["sfreq"] / 5.)
+        peaks = argrelmax(gfp, order=order)
+        if len(peaks) > 10:
+            peaks = peaks[np.sort((-peaks).argsort()[:10])]
```

Comment:
So even if you had 10 peaks in the first 1 second of data and basically
noise for the 9 following seconds, you'd expect "times=peaks" to show 1
peak for each second of data? I guess we differ in expectations there. But
more objectively, I'm not really sure why that's a really useful behavior
given how similar it would be to the "auto" mode that distributes evenly
across time. Maybe auto mode should actually behave that way, looking for
peak values in each tenth of the data.